common/src/main/java/com/wynntils/commands/WynntilsCommand.java

```diff
@@ -79,9 +82,14 @@ private int reload(CommandContext<CommandSourceStack> context) {
             feature.disable();
         }
 
-        WebManager.reset();
+        // Try to reload data from the web. Likely to be broken, see
+        // https://github.com/Wynntils/Artemis/issues/824
         ItemProfilesManager.reset();
-        WebManager.init(); // reloads api urls as well as web manager
+        // reloads api urls as well as web manager
+        UrlManager.reloadUrls();
+        ItemProfilesManager.init();
+        SplashManager.init();
+        WynntilsAccountManager.init();
```

Comment:
Shouldn't we reset the other managers? (I know this is bad as a whole, so maybe we can just ignore this for the time being)

Reply:
So, to be clear, I have copied as good as humanely possible what the old WebManager.reset() did. It held on to a few, but not all, of the "profile" data. These are the ones I have called init on.

Reply:
I thought I was clear enough that the entire thing was a mess that needs to be addressed by https://github.com/Wynntils/Artemis/issues/824 :)

Reply:
Yes, this is bad. Just one question. Do you have plans to address this yourself, or should I be looking into it sometime?

Reply:
I can look into it. I think the entire code base could do with another sweep, which looks more closely at error handling and exceptional conditions.

However.

I'm currently quite fed up with this, I've been working on this for two weeks (not counting the time I was contemplating the design prior to that) and I'd like to take a break. So if you think it needs to be fixed "soon-ish", maybe you should to it.

Reply:
Okay. That is understandable. We will see who gets to this first. I will let you know, if I start working on this (and it would be great if you could do this for me too).

Reply:
Sure, I'll let you know. (In general, it is probably a good idea to assign a bug to yourself, and also perhaps add a comment, if you intend to start solving it)